packages/shopify-checkout/src/client/actions/queryCheckout.ts

```diff
@@ -0,0 +1,19 @@
+import { GqlClient, ShopifyResponse } from '../../checkout-client.types';
+
+export interface CheckoutQueryParams {
+  gqlClient: GqlClient;
+  query: string;
+  variables: Record<string, unknown>;
+}
+export default function queryCheckout({
+  gqlClient,
+  query,
+  variables
+}: CheckoutQueryParams): Promise<ShopifyResponse<
+  Record<string, unknown>
+> | void> {
```

Comment:
Not a big deal, but since we aren't throwing any errors in `queryCheckout` or doing anything else that would prevent us from returning:

```ts
Promise<ShopifyResponse<Record<string, unknown>>>
```
, we could get rid of the ` | void` in the return type:

```suggestion
}: CheckoutQueryParams): Promise<ShopifyResponse<Record<string, unknown>>> {
```